### PR TITLE
dependency updates

### PR DESCRIPTION
**Pull request by wbrunette**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,8 +84,8 @@
     </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-      <version>${org.apache.commons-lang3.version}</version>
+      <artifactId>commons-text</artifactId>
```
Comment (linl33) [inline pom.xml:87]: @wbrunette Why was this changed to `commons-text`? Is it used in `odk-rest-interface`?

**Review by jbeorse — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (wbrunette) [reply pom.xml:87]: yes it's used in odk-rest-interface

Comment (wbrunette) [reply pom.xml:87]: specifically it's used for html escaping
